spec/models/archive_endpoint_spec.rb

```diff
@@ -73,4 +73,82 @@
       expect(described_class.pluck(:endpoint_name).sort).to eq expected_ep_names
     end
   end
+
+  describe '.archive_targets' do
+    let!(:alternate_pres_policy) do
```

Comment:
do we want a factory for preservation policy?

Reply:
would be inclined to do that as another PR, but seems like a good idea

Reply:
#984